src/sknnr/datasets/_base.py

```diff
@@ -66,17 +66,35 @@ def _open_text(module_name: str, file_name: str) -> IO[str]:
     return resources.open_text(module_name, file_name)
 
 
-def _load_csv_data(
+def load_csv_data(
     file_name: str,
+    *,
+    data_module: str = DATA_MODULE,
```

Comment:
I should've done this originally, but this (and `_open_text` and `load_dataset_from_csv_filenames`) should probably be annotated with `str | types.ModuleType`.